cli/tests/node_compat/config.json

```diff
@@ -210,7 +209,6 @@
       "test-child-process-exec-stdout-stderr-data-string.js",
       "test-child-process-exec-timeout-expire.js",
       "test-child-process-exec-timeout-kill.js",
-      "test-child-process-execFile-promisified-abortController.js",
```

Comment:
Can we make this jsonc and comment these out?

Reply:
Done